app/views/events/show.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="container">
   <p>
-    <strong><%= t('.event_status') %> </strong>
-    <%= @event.event_status %>
+    <strong><%= t('.status') %> </strong>
```

Comment:
Create this translation as `Event` model translation. You have that in good place in `yml` file, but here you should use it in different way.

Reply:
Still missing the right translations.